### DISCUSSION TITLE
2.03版在开启vpn的时候似乎会连不上（ios）

### DISCUSSION THREAD

**Discussion by fatsnk** [xiaoyaocz/flutter_dmzj#128] [General]
ios系统连接了vpn之后，2.02是正常的。升级2.03就看不了了，只能退回2.02

**Comment by xiaoyaocz** [comment]
2.0.3将访问域名改至动漫之家最新的idmzj.com，旧域名不确定什么时候会停用，建议设置VPN相关规则
